### DISCUSSION TITLE
ISDA CDM Discussion - vote for a date

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#43] [General]
Hi All,

Further to the email from Jane, we need to get together and discuss the CDM proposal. As you'll have seen from the supplied materials, the specific question the FINOS team would like us to consider is, whether the preferred route forwards is "Rosetta Studio, Legend Studio, or both"

Here are some date options for a potential meeting:

- [A] Mon 7th, 17:00
- [B] Tue 8th, 15:00
- [C] Wed 9th, 13:00 - **Voting closed** - we've picked this option
- [D] Thur 10th, 16:00

(All times in GMT, for New York subtract 5!)

Please reply to this discussion listing the letters for the times you can attend.

**Comment by ColinEberhardt** [comment]
A, B, C, D ;-)

**Comment by brunodom** [comment]
A and B

**Comment by jonfreedman** [comment]
B, C, D

**Comment by maoo** [comment]
C and D

For B, I could join with 30minutes delay

**Comment by jgavronsky** [comment]
B, C are preferred but if no other choice, can do D as well

**Comment by ColinEberhardt** [comment]
Thanks everyone, based on the feedback here, and via email, we'll be going for option C
